tokenizers/src/pre_tokenizers/metaspace.rs

```diff
@@ -44,6 +67,20 @@ impl Metaspace {
             replacement,
             str_rep: replacement.to_string(),
             add_prefix_space,
+            prepend_scheme: PrependScheme::Always, // always prepend for legacy purpose
+        }
+    }
+
+    pub fn new_with_prepend_scheme(
```

Comment:
Kind of syntactic, but making `new`  call `new_with_prepend`  should make everything less error prone (since there's less room for modifying one path without modifying the other.